app/Http/Controllers/SrdController.php

```diff
@@ -40,12 +40,19 @@ public function searchRoutes(Request $request): JsonResponse
             })
                 ->where('maximum_level', '>=', $requestData['requestedLevel']);
         }
+        
+        if (strlen($requestData['destination']) > 4) {
```

Comment:
Could possibly implement this as a ternary for readability - `$exit = strlen($requestData['destination']) == 4 ? null : $requestData['destination']`;

Reply:
Also, would we need to consider VORs? Thinking here in AC West where Strumble (STU) is a valid exit point - so we might want to do != 4 rather than >

Reply:
Fixed